neurolib/optimize/exploration/exploration.py

```diff
@@ -333,3 +402,32 @@ def getRun(self, runId, filename=None, trajectoryName=None, pypetShortNames=True
             pypetTrajectory = pu.loadPypetTrajectory(filename, trajectoryName)
 
         return pu.getRun(runId, pypetTrajectory, pypetShortNames=pypetShortNames)
+
+    def getResult(self, runId):
+        """Returns either a loaded result or reads from disk.
+
+        :param runId: runId of result
+        :type runId: int
+        :return: result
+        :rtype: dict
+        """
+        if hasattr(self, "results"):
+            # load result from either the preloaded .result attribute (from .loadResults)
+            result = self.results[runId]        
+        else:
+            # or from disk if results haven't been loaded yet
+            result = self.getRun(runId)
+        return result
+
+    def info(self):
+        """Print info about the current search.
+        """
+        print(
```

Comment:
I think this is exactly what predefined `__str__(self)` functions are for in classes:)